package.json

```
@@ -17,7 +17,7 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
-    "cordova-common": "^3.1.0",
+    "cordova-common": "4.0.0-nightly.2019.11.8.40cb6cee",
```

If I am correct, you can set the version to a tag (`nightly`) so it will always fetch the latest nightly. This means we wouldn't need to always update the `package.json` as we continue to add changes to common.

```suggestion
    "cordova-common": "nightly",
```

The releaser would need to remember to change this to an official released version before release.

---

I actually deliberately chose to use a fixed nightly to keep our CI tests stable. I think it's actually good having to manually bump the dependency version to get new features. That explicitly shows the dependence on a new feature.